library.gradle

```diff
@@ -10,7 +10,7 @@ android {
         minSdkVersion minVersion
         targetSdkVersion compileVersion
         versionCode 1
-        versionName "3.5.0-SNAPSHOT"
+        versionName "3.5.4-SNAPSHOT"
```

Comment:
I'm a bit confused about this number bump - we haven't released 3.5.0 yet, and the latest release is 3.4.1

Reply:
should this have been "3.4.2" (no snapshot) instead?

Reply:
oh yes, sorry I incrementally increased this when testing because gradle was caching the version and wasn't able to force a re-download of the latest version. `develop` has the subscriber attributes, so this should still be a `SNAPSHOT` version. I will roll back to 3.5.0-SNAPSHOT, which is what we have in develop.